CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 0.7.0 (2022-08-03)
 
+### Bug fixes
```

Comment:
This is not going to be included to version 0.7.0 Can you do something like this?
```suggestion
# Changelog

## Unreleased version

### Bug fixes

* [#122](https://github.com/sarslanoglu/turkish_cities/issues/122): ArgumentError: comparison of Array with Array failed

## 0.7.0 (2022-08-03)

### New features
...
```